sdk/resourcemanager/Azure.ResourceManager.Core/src/Azure.ResourceManager.Core.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is this just a whitspace change?  Can we remove it?

Reply:
it changes to file to utf-8 which i believe is the desired state it was automatically done by the prepare-release.ps1 script